robottelo/ui/base.py

```diff
@@ -539,11 +540,14 @@ def click(self, locator, wait_for_ajax=True,
         :raise: UINoSuchElementError if the element could not be found.
 
         """
-        element = self.wait_until_element(locator, timeout=waiter_timeout)
+        if type(locator).__name__ == 'tuple':
```

Comment:
Not bad idea, but that approach will break the logic.
That should work only for locators: ```element = self.wait_until_element(locator, timeout=waiter_timeout)```
And that should work for both `WebElement` and `None` values: `element = locator`